internal/proxyservice/tcp/pg/plugin.go

```diff
@@ -0,0 +1,45 @@
+package pg
+
+import (
+	"net"
+
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector/tcp"
+)
+
+// NewConnector returns a tcp.Connector which returns an authenticated connection
+// to a target service for each incoming client connection.
+// It is a required method on the tcp.Plugin interface.
+// The single argument passed in is of type connector.Resources. It contains
+// connector-specific config and a logger.
```

Comment:
Nit: formatting.  Hotkey that bad boy. Also I think lines 13/14 aren't needed, ie, no need to re-iterate the interface def of `connector.Resources` in the docs for `NewConnector`